spec/spec_helper_acceptance_local.rb

```diff
@@ -54,7 +54,7 @@ class { 'epel':
           epel_baseurl => "http://osmirror.delivery.puppetlabs.net/epel${::operatingsystemmajrelease}-\\$basearch/RPMS.all",
           epel_mirrorlist => "http://osmirror.delivery.puppetlabs.net/epel${::operatingsystemmajrelease}-\\$basearch/RPMS.all",
         }
-      } elsif $::operatingsystemmajrelease == '8' {
+      } elsif $::operatingsystemmajrelease == '8' or $::operatingsystemmajrelease == '9' {
           class { 'epel':
                 os_maj_release => "7",
                 epel_baseurl => "http://osmirror.delivery.puppetlabs.net/epel7-\\$basearch/RPMS.all",
```

Comment:
Isue seems to come from us still using the deprecated `stahnma/epel`rather than the more currently maintained vox version, have created an issue to track: https://github.com/puppetlabs/puppetlabs-postgresql/issues/1344

Reply:
https://github.com/puppetlabs/puppetlabs-apache/pull/2243

Reply:
Thank's for the quick fix, have rebased on top of it.